plugins/woocommerce/client/blocks/assets/js/blocks/mini-cart/iapi-frontend.ts

```diff
@@ -164,3 +170,156 @@ store(
 	},
 	{ lock: true }
 );
+
+type CartItemContext = {
+	cartItem: CartItem;
+};
+
+store( 'woocommerce/mini-cart-items-block', {
+	state: {
+		// Intended to be used in context of a cart item in wp-each
+		get reduceQuantityLabel() {
+			const { reduceQuantityLabel } = getConfig(
+				'woocommerce/mini-cart-items-block'
+			);
+			const { cartItem } = getContext< CartItemContext >();
+
+			return reduceQuantityLabel.replace( '%s', cartItem.name );
```

Comment:
Maybe? (also in the other getters)

```suggestion
			return reduceQuantityLabel.replace( '%s', state.cartItemName );
```

Reply:
This is basically solved by your commit with derived state for the cart item. 🎉